flopy/export/vtk.py

```diff
@@ -1260,50 +1265,80 @@ def add_cell_budget(
             self.add_transient_array(d, name, masked_values)
             self.__transient_output_data = True
 
-    def _set_modpath_point_data(self, points, d):
+    def _set_particle_track_data(self, points, lines=None, arrays=None):
         """
-        Method to build the vtk point geometry and set arrays for
-        modpath pathlines
+        Build VTK data structures for particle positions, pathlines, and metadata
 
         Parameters
         ----------
-        points : list
+        points : list or array-like
```

Comment:
Numpydoc uses [`array_like`](https://numpydoc.readthedocs.io/en/latest/format.html#other-points-to-keep-in-mind) for these types:
```suggestion
        points : list or array_like
```
same for L1276